.github/workflows/integration.yaml

```diff
@@ -56,6 +56,7 @@ jobs:
          run: |
            pip install -U setuptools wheel
            pip install -r dev_requirements.txt
+           [ $(docker ps -aq | wc -l) -ne 0 ] && docker rm -f $(docker ps -aq)
```

Comment:
This shouldn't be necessary in GH Actions. What did you run into here?

Reply:
It is one of the weird rare failure cases with gh.  I agree that it shouldn't be needed, but what I ran into was that the redis container couldn't start because the port was in use.  I have no idea if this fixed it, but I didn't see it again.

Reply:
removed.  tests pass.